.github/renovate.json5

```
@@ -45,4 +45,6 @@
   "enabledManagers": [
     "tekton",
   ],
+  labels: ["auto-merge"],
+  "platformAutomerge": true
```

[nit] An advantage of json5 is that it allows trailing comma. The one helps have less changed lines in diffs. If we keep this attribute, please add one
```suggestion
  "platformAutomerge": true,
```